app/static/js/initialize_map.js

```diff
@@ -0,0 +1,119 @@
+geocoder = new google.maps.Geocoder();
+address = document.getElementById("address");
+
+// Global array for map markers
+var markers = []
+
+// Global Marker Wrappers and Map
+var globalMarkerWrappers = null;
+var globalMap = null;
+
+// Initial map center coordinates
+INITIAL_CENTER_LAT = 39.952;
+INITIAL_CENTER_LONG = -75.195;
+initial_coords = new google.maps.LatLng(INITIAL_CENTER_LAT, INITIAL_CENTER_LONG);
+
+// Initial map zoom value
+ZOOM = 17;
+
+// Bounds for the Date Slider. End date is current date.
+var BOUNDS_MIN = new Date(2015, 0, 1);
```

Comment:
Actually ignore this comment because this date shows up in the dateslider

I think min date should be the min date of all the reports and max date should be the max date of all reports